api.js

```diff
@@ -180,6 +186,25 @@ Api.prototype._prefixTitle = function (file) {
 	return prefix;
 };
 
+Api.prototype._restartTimer = debounce(function () {
```

Comment:
The point of the latest update is to create less timers. That's why `debounce` should be on `_restartTimer()`.

> Also, you shouldn't use debounce on a prototype method, as it will debounce across every instance and only call on the most recent.

Agree.